.github/workflows/test-linux.yml

```diff
@@ -50,8 +55,8 @@ jobs:
       - name: Build
         run: cargo build
 
-      - name: Run unit tests
+      - name: Run tests
         env:
+          NEXTEST_PROFILE: "ci"
           AIR_LOG_LEVEL: trace
-        # `--nocapture` to see our own `tracing` logs
-        run: cargo test -- --nocapture
+        run: cargo nextest run
```

Comment:
nextest will show all stdout and stderr for any failing tests, so I don't think we really need `-- --nocapture` anymore.

The nextest equivalent is `--no-capture`, but that also (smartly) forces sequential tests to prevent live output from interleaving between tests and I'd rather not sacrifice parallelism just to see output on passing tests.